pyqtgraph/graphicsItems/LabelItem.py

```diff
@@ -55,7 +55,7 @@ def setText(self, text, **args):
         if color is None:
             color = getConfigOption('foreground')
         color = fn.mkColor(color)
-        optlist.append('color: ' + color.name())
+        optlist.append('color: ' + color.name(QtGui.QColor.HexArgb))
```

Comment:
I think this needs to be `QtGui.QColor.NameFormat.HexArgb`

PyQt6 will likely fail with the enum namespace lookup.